CMakeLists.txt

```diff
@@ -47,8 +47,6 @@ if( NOT ELASTIX_BUILD_EXECUTABLE )
   #set( CMAKE_C_VISIBILITY_PRESET hidden )
   #set( CMAKE_CXX_VISIBILITY_PRESET hidden )
 
-  mark_as_advanced( BUILD_SHARED_LIBS )
-  option( BUILD_SHARED_LIBS "Build shared libraries?" OFF )
```

Comment:
Thanks Matt. I'm sorry I haven't been able to reproduce the CMake warning so far, using CMake GUI 3.15.4 on Windows 10. So the default of BUILD_SHARED_LIBS is `OFF`, right? Then I guess indeed, these lines may go.

Reply:
> So the default of BUILD_SHARED_LIBS is OFF, right?

Yes, correct :+1: